### ISSUE TITLE
[0.56.0] Unable to resolve module `AccessibilityInfo` when init App on WINDOWS

### ISSUE THREAD

**Issue by QinGuan2004** [facebook/react-native#20689]
## Environment
Windows 10
React Native 0.56.0
React 16.4.1
[skip envinfo]

## Description
Unable to resolve module `AccessibilityInfo` when loading app 

## Reproducible Demo
On *WINDOWS*, 
```$ react-native init```
```$ react-native run-android```

Error should appear when app is loading from packager.
Issue has been reported before on React Native v0.4x

**Comment by hramos** [comment]
Duplicate of #20353

**Comment by fantasy525** [comment]
#20707 I got same issues ,and I have enviroment Infos, @react-native-bot

**Comment by QinGuan2004** [comment]
This is duplicate but issue was never solved properly @hramos is there usable solution

**Comment by hramos** [comment]
It's still a duplicate - you can subscribe to #20353 for updates.
